DDR4 OC Guide.md

```diff
@@ -222,66 +226,72 @@ As far as I know, tCL, tRCD, tRP and possibly tRFC can (or can not) see voltage
+* It may be safe to daily 1.60V as there are kits on the [B550 Unify-X QVL](https://www.msi.com/Motherboard/support/MEG-B550-UNIFY-X#support-mem-20) rated for 1.60V. B-Die, 8Gb Rev. E, DJR and 16Gb Rev. B *should* be fine at running 1.60V daily, though it's recommended to have active airflow. Higher voltages leads to higher temperatures and high temperatures can themselves lower the threshold for what voltages are considered safe.
   
 ### Ranking
 * Below is how most of the common ICs rank in terms of frequency and timings.
-  1. Samsung 8Gb B-die
-  2. Micron 16Gb Rev. B
-  3. Micron 8Gb Rev. E, Samsung 4Gb E-die, Hynix 8Gb DJR, Nanya 8Gb B-die
-  4. Hynix 8Gb CJR, Samsung 8Gb D-die 
-  6. Micron 16Gb Rev. E, Samsung 8Gb C-die
-  7. Hynix 8Gb AFR
-  8. Samsung 4Gb D-die
-  9. Hynix 8Gb MFR 
-  10. Samsung 4Gb S-die
-  * Based off [buildzoid's ranking](https://www.reddit.com/r/overclocking/comments/8cjla5/the_best_manufacturerdie_of_ddr_ram_in_order/dxfgd4x/).
-  * Note that the ICs that have the same rank are in no particular order.
+  | Tier | ICs | Description |
+  | :-:  | :-: | :--:        |
+  | S | Samsung 8Gb B-Die | Best DDR4 IC for all round performance |
+  | A | Hynix 8Gb DJR, Micron 8Gb Rev. E<sup>1</sup>, Micron 16Gb Rev. B | Top Performing ICs. Known not to clockwall and generally scale with voltage. |
+  | B | Hynix 8Gb CJR, Samsung 4Gb E-Die, Nanya 8Gb B-Die | High end ICs with the ability to run high frequencies with good timings. |
+  | C | Hynix 8Gb JJR, Hynix 16Gb MJR, Hynix 16Gb CJR, Micron 16Gb Rev. E, Samsung 8Gb D-Die, Nanya 8Gb A-Die | Decent ICs with good performance and decent frequency scaling.|
+  | D | Hynix 8Gb AFR, Hynix 8Gb BFR, Micron 8Gb Rev. B, Micron 8Gb Rev. H, Samsung 8Gb C-Die, Samsung 4Gb D-Die | Low end ICs commonly found in average cheap kits. Most are EOL and no longer relevant. 
+  | F | Hynix 8Gb MFR, Micron 4Gb Rev. A, Samsung 4Gb S-Die, Nanya 4Gb A-Die, Nanya 8Gb C-Die | Terrible ICs unable to reliably attain even the highest standard of the base JEDEC Specification.|
+  * Partially based off [Buildzoid's older ranking](https://www.reddit.com/r/overclocking/comments/8cjla5/the_best_manufacturerdie_of_ddr_ram_in_order/dxfgd4x/). Some ICs are not included in this list due to age of the post.
+  * <sup>1</sup> 8Gb Rev. E is extremely varied. The chips and various revisions that all fall under 8Gb Rev. E range from an IC that is comfortable overclocking with above average timings to an extremely strong one, being the world record holding IC and being known to get certain timings insanely tight.
```

Comment:
Main differences are how tight tRCD can go and how high you can OC without VTT adjustment. The newer revisions (C9BKV, C9BLL, etc) can do tighter tRCD and higher frequency but everything else is similar.

* [3600 16-20-18 1.35V](https://www.reddit.com/r/Amd/comments/bpybh1/ddr43600_162018_on_4_dimms_with_ryzen_6400_karhu/)
* [3733 16-17-17 1.40V](https://www.reddit.com/r/overclocking/comments/bqnq0m/micron_edie_3733mhz_cas16_14v/)
* [3600 14-20-15 1.45V](https://www.reddit.com/r/Amd/comments/be9fou/some_more_fun_with_1600x_and_cheap_micron_edies/)

Reply:
theres also a slight difference to tRFC on average. 300ns min down to 280 min.

Reply:
addressed